src/components/calculator.js

```diff
@@ -1,63 +1,64 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 import updateObject from '../logic/parseUpdate';
 // eslint-disable-next-line react/prefer-stateless-function
-export default class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
 
-  handleClick = (buttonName) => {
-    const destr = this.state;
-    const updatedObject = updateObject(destr, calculate(destr, buttonName));
-    this.setState(updatedObject);
+export default function Calculator() {
+  const [nextRes, setNext] = useState(null);
+  const [operationRes, setOperation] = useState(null);
+  const [totalRes, setTotal] = useState(null);
+
+  function updateState(object) {
+    setNext(object.next);
+    setOperation(object.operation);
+    setTotal(object.total);
```

Comment:
- [ ] As we have learned ES6 syntax why not we use it here too? Please change this to ES6 syntax.